src/Database/Eloquent/Relations/BelongsToMany.php

```diff
@@ -119,7 +119,7 @@ protected function baseAttachRecord($id, $timed)
      *
      * @return \Illuminate\Database\Query\Builder
      */
-    protected function newPivotQuery()
+    public function newPivotQuery()
```

Comment:
apps using 5.* would break right?

Reply:
Correct. I've updated the Laravel version required by this package to be `^6.0`